plugins/node/opentelemetry-instrumentation-ioredis/src/utils.ts

```diff
@@ -16,34 +16,10 @@
 
 import { Span, SpanStatusCode } from '@opentelemetry/api';
 import { DbStatementSerializer } from './types';
+import { defaultDbStatementSerializer as redisSerializer } from '@opentelemetry/redis-common';
 
-/**
- * List of regexes and the number of arguments that should be serialized for matching commands.
- * For example, HSET should serialize which key and field it's operating on, but not its value.
- * Setting the subset to -1 will serialize all arguments.
- * Commands without a match will have their first argument serialized.
- *
- * Refer to https://redis.io/commands/ for the full list.
- */
-const serializationSubsets = [
-  {
-    regex: /^ECHO/i,
-    args: 0,
-  },
-  {
-    regex: /^(LPUSH|MSET|PFA|PUBLISH|RPUSH|SADD|SET|SPUBLISH|XADD|ZADD)/i,
-    args: 1,
-  },
-  {
-    regex: /^(HSET|HMSET|LSET|LINSERT)/i,
-    args: 2,
-  },
-  {
-    regex:
-      /^(ACL|BIT|B[LRZ]|CLIENT|CLUSTER|CONFIG|COMMAND|DECR|DEL|EVAL|EX|FUNCTION|GEO|GET|HINCR|HMGET|HSCAN|INCR|L[TRLM]|MEMORY|P[EFISTU]|RPOP|S[CDIMORSU]|XACK|X[CDGILPRT]|Z[CDILMPRS])/i,
-    args: -1,
-  },
-];
+export const defaultDbStatementSerializer: DbStatementSerializer =
+  redisSerializer;
```

Comment:
No need to import and export in utils.
You can directly 
```js
// instrumentation.ts
import { defaultDbStatementSerializer } from '@opentelemetry/redis-common';
```

in `instrumentation.ts` and remove it from `utils.ts`